build/azure-pipelines/sql-dacfx-build.yml

```diff
@@ -0,0 +1,95 @@
+resources:
+  containers:
+  - container: linux-x64
+    image: sqltoolscontainers.azurecr.io/linux-build-agent:2
+    endpoint: ContainerRegistry
+
+jobs:
+- job: Compile
+  pool:
+    vmImage: 'Ubuntu-16.04'
+  container: linux-x64
+  steps:
+  - script: |
+      set -e
+      echo "##vso[build.addbuildtag]$(VSCODE_QUALITY)"
+    displayName: Add Quality Build Tag
+  - template: sql-product-compile.yml
+
+- job: macOS
+  condition: and(succeeded(), eq(variables['VSCODE_BUILD_MACOS'], 'true'), ne(variables['VSCODE_QUALITY'], 'saw'))
+  pool:
+    vmImage: macOS-latest
+  dependsOn:
+  - Compile
+  steps:
+  - template: darwin/sql-product-build-darwin.yml
+  timeoutInMinutes: 180
+
+- job: macOS_Signing
+  condition: and(succeeded(), eq(variables['VSCODE_BUILD_MACOS'], 'true'), eq(variables['signed'], true), ne(variables['VSCODE_QUALITY'], 'saw'))
+  pool:
+    vmImage: macOS-latest
+  dependsOn:
+  - macOS
+  steps:
+  - template: darwin/sql-product-build-darwin-signing.yml
+  timeoutInMinutes: 60
+
+- job: Linux
+  condition: and(succeeded(), eq(variables['VSCODE_BUILD_LINUX'], 'true'))
+  pool:
+    vmImage: 'Ubuntu-16.04'
+  container: linux-x64
+  dependsOn:
+  - Compile
+  steps:
+  - template: linux/sql-product-build-linux.yml
+    parameters:
+      extensionsToUnitTest: ["admin-tool-ext-win", "agent", "azdata", "azurecore", "cms", "dacpac", "import", "schema-compare", "notebook", "resource-deployment", "machine-learning", "sql-database-projects", "data-workspace"]
+  timeoutInMinutes: 70
+
+- job: LinuxWeb
+  condition: and(succeeded(), eq(variables['VSCODE_BUILD_WEB'], 'true'), ne(variables['VSCODE_QUALITY'], 'saw'))
+  pool:
+    vmImage: 'Ubuntu-16.04'
+  container: linux-x64
+  variables:
+    VSCODE_ARCH: x64
+  dependsOn:
+  - Compile
+  steps:
+  - template: web/sql-product-build-web.yml
+
+# - job: Docker
+#   condition: and(succeeded(), eq(variables['VSCODE_BUILD_DOCKER'], 'true'))
+#   pool:
+#     vmImage: 'Ubuntu-16.04'
+#   container: linux-x64
+#   dependsOn:
+#   - Linux
+#   steps:
+#   - template: docker/sql-product-build-docker.yml
+
+- job: Windows
+  condition: and(succeeded(), eq(variables['VSCODE_BUILD_WIN32'], 'true'))
+  pool:
+    vmImage: VS2017-Win2016
+  dependsOn:
+  - Compile
+  steps:
+  - template: win32/sql-product-build-win32.yml
+  timeoutInMinutes: 70
+
+- job: Windows_Test
```

Comment:
Why are you making a separate pipeline file for this vs just using env vars to control which parts are ran? Doing it like this means that if we make changes to the other files then we have to remember to also update this or it may break...

Reply:
I'm just playing around with pipelines here. I might revert these files

Reply:
but yeay, trying to see if I can create a separate pipelines for DacFx itnegration tests

Reply:
Cool, feel free to ping me if you have questions about how all this is set up